tempesta_fw/sock_srv.c

```diff
@@ -471,6 +438,10 @@ tfw_sock_srv_start(void)
 static void
 tfw_sock_srv_stop(void)
 {
+	/*
+	 * Connections list is read-only at run time for now, so no need
+	 * to synchronize the list access or disable softirqs.
+	 */
```

Comment:
Yes, we do this in `ss_synchronize()` called from `tfw_cfg_stop()` (it surely must be moved from Cfg module as well in future). In particular we're waiting until all connection active counters are zeroed on all the CPUs.